src/sentry/utils/security/orgauthtoken_jwt.py

```diff
@@ -0,0 +1,31 @@
+from datetime import datetime
+from uuid import uuid4
+
+from django.conf import settings
+
+from sentry.utils import jwt
+
+
+def generate_token(org_slug: str):
+    jwt_payload = {
+        "iss": "sentry.io",
+        "iat": datetime.utcnow(),
+        "nonce": uuid4().hex,
+        "sentry_site": settings.SENTRY_OPTIONS["system.url-prefix"],
+        "sentry_org": org_slug,
```

Comment:
I think you'll also need the organizations 'root' URL as in the future, we'll have multiple regions (with different API urls). While customer traffic can continue using `https://sentry.io/api/0/**` there will be latency overhead (and cost to us) so we should encourage using regional domains instead.

Reply:
is this different from `"sentry_site": settings.SENTRY_OPTIONS["system.url-prefix"],`? that is what this is designed to do (but honestly I am not 100% sure what fields are what there, @mitsuhiko pointed me to this settings field)

So to be clear, the only point of the `sentry_site` field is exactly what you mentioned, if we should store something else in there, happy to be pointed to it!

Reply:
In the future, organizations will have a couple URLs:

- `sentryUrl` which is our root domain and is the same as `system.url-prefix`.  ex. `sentry.io`
- `organizationUrl` which is domain for the organization's UI. ex `acme.sentry.io`
- `regionUrl` Which is the domain that the organization's API endpoints are available on. ex `us.sentry.io`

Any API requests sent to `sentryUrl` will be routed to the correct region, but will suffer a latency penalty (because we're proxying the request to the region) and the request will go to the US (this can matter for EU customers).

For this scenario, I think you'll want both the `sentryUrl` and `regionUrl` available in the JWT. There are a small number of API endpoints that are only available on `sentryUrl` (users, integrations, sentry apps).

Reply:
where do I get `regionUrl` from? (I left a comment in the RFC as well to make sure this is aligned - I'd go with `sentry_url` and `sentry_region_url` then)